interface.js

```diff
@@ -20,7 +20,10 @@
 
 'use strict';
 
+var color = require('ansi-color').set;
```

Comment:
chalk is very well done and I recommend it over ansi-color https://www.npmjs.com/package/chalk